app/jobs/appointment_reminder_job.rb

```diff
@@ -4,8 +4,7 @@ class AppointmentReminderJob < ApplicationJob
   def perform(appointment)
     ReminderActivity.create!(
       appointment: appointment,
-      owner: appointment.guider,
-      message: 'sent a reminder'
```

Comment:
For completion sake should we change the existing messages for this activity to `''`?